packages/forms/src/lib/timepicker/README.md

```diff
@@ -0,0 +1,82 @@
+# @acpaas-ui/ngx-components/forms
+
+The timepicker package provides an easy way to pick a time by providing two fields: 1 for the hour value and 1 for the minutes value. You can pick a time either by choosing a slot in the dropdown or by typing in the autocomplete field.
+
+## Usage
+
+```typescript
+import { TimepickerModule } from '@acpaas-ui/ngx-components/timepicker'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() hoursPlaceholder: string;` | - | The placeholder used in the hour picker (optional, default HH). |
+| `@Input() minutesPlaceholder: string;` | - | The placeholder used in the minutes picker, (optional, default MM). |
+| `@Input() hasError: boolean;` | - | Add has-error class to input fields inside timepicker, (optional, default false). |
+| `@Input() size: TimepickerInputSize;` | - | enum: sizes of the timepicker input fields instide the timepicker, (optional, default auto). |
```

Comment:
You mean 'Enum sizes of ' ... 'inside'?
And shouldn't the default values be mentioned in the column `Default value`?